pkg/kor/filter_options.go

```diff
@@ -0,0 +1,126 @@
+package kor
+
+import (
+	"errors"
+	"time"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+)
+
+// FilterOptions represents the flags and options for filtering unused Kubernetes resources, such as pods, services, or configmaps.
+// A resource is considered unused if it meets the following conditions:
+//   - Its age (measured from the last modified time) is within the range specified by MinAge and MaxAge flags.
+//     If MinAge or MaxAge is zero, no age limit is applied.
+//   - Its size (measured in bytes) is within the range specified by MinSize and MaxSize flags.
+//     If MinSize or MaxSize is zero, no size limit is applied.
+//   - It does not have any labels that match the ExcludeLabels flag. The ExcludeLabels flag supports '=', '==', and '!=' operators,
+//     and multiple label pairs can be separated by commas. For example, -l key1=value1,key2!=value2.
+type FilterOptions struct {
+	// MinAge in the minimum age of the resources to be considered unused
+	MinAge time.Duration
```

Comment:
Any chance we can add support for days as well?

Reply:
added